### PR TITLE
Minor update to the Dashboard page to link to the newly created admin pages

### PR DESCRIPTION
**Pull request by fitore-cds**
# Summary | Résumé

My humble contribution to the dashboard page as I'm catching up to the sandbox work :-)

### CODE REVIEW

**Comment by brdunfield** [comment]
PR app isn't working. Is that expected? Not sure the state of Heroku with sandbox apps

File: pages/api/submit.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ const previewNotify = async (form, fields) => {
   );
 
   const emailBody = await convertMessage({ form, responses: fields });
-  const messageSubject = `${form.emailSubjectEn ? form.emailSubjectEn : form.titleEn} Submission`;
+  const messageSubject = form.emailSubjectEn ? form.emailSubjectEn : form.titleEn;
```
Comment (brdunfield) [inline pages/api/submit.js:76]: Was this change intentional? Looks unrelated to the admin page so just checking :)

File: pages/admin/index.tsx [hunk 2]
```diff
@@ -17,15 +18,36 @@ const AdminWelcome: React.FC<AdminWelcomeProps> = (props: AdminWelcomeProps) =>
...
+
+          <Button type="button" onClick={() => signOut()}>
+            {t("button.logout")}
+          </Button>
+        </div>
+
+        <div className="w-60">
+          <h3 className="gc-h3">Create Forms</h3>
+          <p>
+            <Link href="/admin/upload">View Form templates</Link>
```
Comment (brdunfield) [inline pages/admin/index.tsx:39]: This should probably say "Upload Form Templates" instead of "View"

File: pages/admin/index.tsx [hunk 3]
```diff
@@ -17,15 +18,36 @@ const AdminWelcome: React.FC<AdminWelcomeProps> = (props: AdminWelcomeProps) =>
...
+          </p>
+        </div>
+        <div className="w-60">
+          <h3 className="gc-h3">View Existing Forms</h3>
+          <p>
+            <Link href="/admin/view-templates">View Form templates</Link>
+          </p>
+          <p>
+            <Link href="/admin/vault">View Form submissions</Link>
```
Comment (brdunfield) [inline pages/admin/index.tsx:48]: this doesn't technically exist yet but it will soon so I don't have a problem leaving it here

File: pages/api/submit.js (hunk 1 above)
Comment (bryan-robitaille) [reply pages/api/submit.js:76]: Yes, that was part of another PR that got merged into Main to support the Transport Safety Board requested changes.  It aligns with what was done on the main branch.

Comment (fitore-cds) [reply pages/api/submit.js:76]: Yes, I snuck this in - it's unrelated but I keep forgetting to update. Thanks both :-)

File: pages/admin/index.tsx (hunk 2 above)
Comment (fitore-cds) [reply pages/admin/index.tsx]: Good catch, thank you! Fixed :-)

**Review by brdunfield — APPROVED** [review]
Looks great! Thank you! 🦑
